matchms/exporting/metadata_export.py

```diff
@@ -38,6 +39,7 @@ def export_metadata_as_json(spectrums: List[Spectrum], filename: str,
     identifier:
         Identifier used for naming each spectrum in the output file.
     """
+    spectrums = filter_empty_spectrums(spectrums)
```

Comment:
Maybe we could make a start here and switch to `spectra`?
```suggestion
    spectra = filter_empty_spectra(spectra)
```

Reply:
This won't work consistently for all code because we have too many instances of `spectrums`, in particular also in function/method names and parameters names. But I feels weird to have both spectrums and spectra in the same code block.

Reply:
Okay, I'll change local variables. However I wont touch the method signature of `export_metadata_as_json` to not break dependencies till tackling #687